ml-agents-envs/mlagents_envs/side_channel/stats_side_channel.py

```diff
@@ -35,6 +38,7 @@ def __init__(self) -> None:
     def on_message_received(self, msg: IncomingMessage) -> None:
         """
         Receive the message from the environment, and save it for later retrieval.
+
```

Comment:
Please remove the extra whitespaces that were added in the docstrings.

Reply:
the extra spaces are necessary for PyCharm

Reply:
Please try to find the settings to disable them; I use PyCharm as do several other team members, and this hasn't been a problem before. At a minimum, you need to revert files that only contain the whitespace changes:
* ml-agents/mlagents/trainers/ppo/trainer.py
* ml-agents/mlagents/trainers/torch/components/reward_providers/gail_reward_provider.py
* ml-agents/mlagents/trainers/trainer/rl_trainer.py

On the other hand, if you know of a way to automate this from the command line (and enforce that the `:param ...` names match the args), I'd love that in another PR.